### PR TITLE
Bootstrap as class

### PR DESCRIPTION
**Pull request by northys**
I've changed bootstrap to use class same as nette/sandbox does. It looks like it works.

```
northys at northys-fedora in ~/c/c/apitte-skeleton (bootstrap_class|✔)
» bin/console
Console Tool

Usage:
  command [options] [arguments]

Options:
  -h, --help            Display help for the given command. When no command is given display help for the list command
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Available commands:
  help                            Display help for a command
  list                            List commands
 dbal
  dbal:import                     Import SQL file(s) directly to Database.
  dbal:reserved-words             Checks if the current database contains identifiers that are reserved.
  dbal:run-sql                    Executes arbitrary SQL directly from the command line.
 doctrine
  doctrine:fixtures:load          Load data fixtures to your database.
 migrations
  migrations:diff                 [diff] Generate a migration by comparing your current database to your mapping information.
  migrations:execute              [execute] Execute a single migration version up or down manually.
  migrations:generate             [generate] Generate a blank migration class.
.......

northys at northys-fedora in ~/c/c/apitte-skeleton (bootstrap_class|✔)
» php www/index.php 
{"status":"error","message":"Client authentication failed","code":401}⏎
```

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -2,7 +2,11 @@
 <?php
 
 /** @var Nette\DI\Container $container */
-$container = require_once __DIR__ . '/../app/bootstrap.php';
+$container = require __DIR__ . '/../vendor/autoload.php';
+
+// Let bootstrap create Dependency Injection container.
+$configurator = \App\Bootstrap::boot();
```
Comment (f3l1x) [inline bin/console:8]: Tiny change.

`\App\Bootstrap::boot()` => `App\Bootstrap::boot()`

File: www/index.php [hunk 2]
```diff
@@ -1,11 +1,10 @@
 <?php declare(strict_types = 1);
 
-// Uncomment this line if you must temporarily take down your site for maintenance.
-//require 'maintenance.php';
-//die();
+require __DIR__ . '/../vendor/autoload.php';
 
 // Let bootstrap create Dependency Injection container.
-$container = require_once __DIR__ . '/../app/bootstrap.php';
+$configurator = \App\Bootstrap::boot();
```
Comment (f3l1x) [inline www/index.php:6]: Tiny change.

`\App\Bootstrap::boot()` => `App\Bootstrap::boot()`

**Review by f3l1x — CHANGES_REQUESTED** [review]

**Comment by f3l1x** [comment]
I like it, just change `\App\...` => `App\..`.

Can you please create same change to other skeletons?

https://github.com/contributte?q=skeleton&type=&language=&sort=

**Comment by northys** [comment]
@f3l1x done
1) removed root namespace
2) removed `$container` in `/bin/console`
3) added `strict_types` in `/bin/console`

**Comment by f3l1x** [comment]
Excellent. 👍  More PR's are welcomed ;-)

**Pull request merged**